Documentation/WritingReST/Reference/Code/Confval.rst

```diff
@@ -28,49 +27,175 @@ Each configuration value name may only be used once. In large references
 with different contexts you can define individual configuration schemas for
 each context.
 
+..  contents:: Table of contents
+
+..  _rest-confval-examples:
+
 Examples
 ========
 
 Required configuration value
 ----------------------------
 
 ..  confval:: label
-
-    :Required: true
+    :name: some-unique-label
+    :required: true
```

Comment:
It does not make sense which value you set. Set it to `0`, `false`, Sphinx native `False` this directive will always show `true`. Don't know what's wrong here. May you have a look?
Else: If `required` is not mandatory, the `required` directive has to be removed completely.

Reply:
I will investigate

Reply:
If it is so I would say its a bug and I will solve it in the render-guides